.clj-kondo/config/modules/config.edn

```diff
@@ -1969,8 +1982,13 @@
   {:team "Querying"
    :api  #{metabase-enterprise.transforms.api
            metabase-enterprise.transforms.core
-           metabase-enterprise.transforms.init}
-   :uses #{api
+           metabase-enterprise.transforms.init
+           metabase-enterprise.transforms.instrumentation
```

Comment:
This feels wrong as part of the public API.

Perhaps the code can be moved somewhere more generic?

Or just moved to our python module for now, if nothing else uses it yet.